tests/e2e/unified_test.go

```diff
@@ -606,6 +606,11 @@ func runScenario(ctx context.Context, t *testing.T, testPause bool, fixture reso
 
 					// Specific to BigQuery
 					addSetStringReplacement(".access[].userByEmail", "user@google.com")
+					for _, event := range events {
+						if event.Request.Method == "PATCH" && strings.Contains(event.Request.URL, "bigquery.googleapis.com/bigquery/v2/") {
+							event.Request.ReplaceHeader("If-Match", "1111111")
```

Comment:
IIUC, it is more readable to set this value to "abcdef0123A=", which is the Etag from previous read.